package.json

```diff
@@ -11,8 +11,7 @@
     "docs": "yuidoc",
     "docs-deploy": "yuidoc && ghp-import -r origin -p -b gh-pages docs/",
     "start": "ember server",
-    "test": "ember test",
-    "check-style": "./node_modules/jscs/bin/jscs ."
+    "test": "./node_modules/jscs/bin/jscs . && ember test"
```

Comment:
You can still run them separately, but travis enforces style checking, so it's important to have the same command available locally to ensure tests are passing. I've seen multiple instances where myself and others have:  
1. pushed a commit
2. had to wait for travis to run
3. the build fails because check-style wasn't run
4. fix the style errors and run check-style again locally
5. push another commit and wait for it to pass in travis

We could avoid those 5 steps by enforcing the check-style up front. If you don't want to run check-style right away, you can just run ember test. However, `npm test` should run all tests that you would expect to have in the CI environment.